com.unity.ml-agents.extensions/Runtime/Match3/Match3Sensor.cs

```diff
@@ -103,14 +113,14 @@ public ObservationSpec GetObservationSpec()
         /// <inheritdoc/>
         public int Write(ObservationWriter writer)
         {
-            if (m_Board.Rows != m_Rows || m_Board.Columns != m_Columns || m_Board.NumCellTypes != m_NumCellTypes)
-            {
-                Debug.LogWarning(
-                    $"Board shape changes since sensor initialization. This may cause unexpected results. " +
-                    $"Old shape: Rows={m_Rows} Columns={m_Columns}, NumCellTypes={m_NumCellTypes} " +
-                    $"Current shape: Rows={m_Board.Rows} Columns={m_Board.Columns}, NumCellTypes={m_Board.NumCellTypes}"
-                );
-            }
+            // if (m_Board.Rows != m_Rows || m_Board.Columns != m_Columns || m_Board.NumCellTypes != m_NumCellTypes)
```

Comment:
Disabled for now, might return later (depending on how I handle multiple board sizes)